frontend/__tests__/src/components/Metrics/ConfigStep/Board.test.tsx

```diff
@@ -89,13 +89,6 @@ describe('Board', () => {
     await waitFor(() => {
       expect(getByText(BOARD_TYPES.CLASSIC_JIRA)).toBeInTheDocument()
     })
-
```

Comment:
Could we rename this test to make it more accurately described? Btw, I don't think there is big differences between using `mouseDown` and `click`, could we only use `click` here?